dss/config.py

```diff
@@ -511,14 +511,16 @@ def __init__(
             storage_schema: str,
             blobstore_class: typing.Type[BlobStore],
             hcablobstore_class: typing.Type[HCABlobStore],
-            flashflood_prefix_envvar: str,
+            flashflood_prefix_read_envvar: str,
```

Comment:
Mind adding in some comments about the purpose of separating the read/write prefixes?  I'm not entirely clear.

Reply:
Not sure this is the appropriate place. Maybe this documentation should live in the script that performs a flashflood reindex (future PR).